renderer/reducers/peripherals.ts

```diff
@@ -32,12 +30,10 @@ export const peripherals = (state: PeripheralState = initialPeripheralState, act
       (action.peripherals ?? []).forEach((peripheral: Peripheral) => {
         if (peripheral.name === consts.PeripheralTypes.BatteryBuzzer) {
           const batteryParams = peripheral.params;
+          //since batterysafety is not included in runtimestatus, have to extract the value from params
           if (batteryParams[IS_UNSAFE] && batteryParams[IS_UNSAFE].bval) {
             nextState.batterySafety = batteryParams[IS_UNSAFE].bval!;
```

Comment:
However batterysafety is not included in the runtime status thus extracted manually.